src/index.html

```diff
@@ -17,16 +17,32 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <div class="inputs">
+      <form
+        class="inputs__form"
+        action="index.html"
+        data-qa="keypress"
+      >
+        <input
+          class="inputs__bigInput"
+          type="text"
+          data-qa="big"
+          placeholder="Try “Los Angeles“"
+        />
+      </form>
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+      <form
+        class="inputs__form"
+        action="index.html"
```

Comment:
Similar to the previous form, the `action` attribute is set to 'index.html'. Ensure this is the desired behavior, or update it to point to the appropriate URL or script for handling form submissions.